core/src/main/scala/org/apache/spark/serializer/KryoSerializer.scala

```diff
@@ -187,14 +187,18 @@ class KryoSerializer(conf: SparkConf)
       "org.apache.spark.mllib.linalg.Matrix",
       "org.apache.spark.mllib.linalg.DenseMatrix",
       "org.apache.spark.mllib.linalg.SparseMatrix",
+      "org.apache.spark.mllib.regression.LabeledPoint",
+      "org.apache.spark.mllib.clustering.VectorWithNorm",
       "org.apache.spark.ml.linalg.Vector",
       "org.apache.spark.ml.linalg.DenseVector",
       "org.apache.spark.ml.linalg.SparseVector",
       "org.apache.spark.ml.linalg.Matrix",
       "org.apache.spark.ml.linalg.DenseMatrix",
       "org.apache.spark.ml.linalg.SparseMatrix",
       "org.apache.spark.ml.feature.Instance",
-      "org.apache.spark.ml.feature.OffsetInstance"
+      "org.apache.spark.ml.feature.OffsetInstance",
+      "org.apache.spark.ml.feature.LabeledPoint",
+      "org.apache.spark.ml.tree.impl.TreePoint"
```

Comment:
I recommend to write these items in alphabet order, so we can easily check whether it miss some item and easier to add more items in the future.